README.md

```diff
@@ -338,6 +345,25 @@ The options `-h`, `-i`, `-r`, `-f` are exclusive. The options `-s` and `-k` are
 at the LPC (where each interactive node has its own scheduler). The script can ssh to each node and run itself to modify the jobs
 on that node (because each scheduler can only be accessed for write operations from its respective node).
 
+The option `-X` will resubmit the jobs (just like `-s`), except that it will tell the job to get its input file from a specific site based on the list of sites where that file
+is located and some user preferences. Therefore, the options `-X` and `-s` are exclusive, as are the options `-X` and `-k`. The options `-C`, `-D`, `-K`, `-L`, `-U`, and `-V`
+are only applicable when using the option `-X`. There are two ways in which the program can find the appropriate input file(s) for the job:
+1. specify an HTCondor ClassAd name, where the ClassAd contains a comma separated list of files (`-C`).
+2. specify the location of a set of log files and a key with which to parse the logs files (`-L`/`-K`). It is expected that the log will contain a single line with a comma separated list of files.
```

Comment:
add two spaces at the end of this line so it doesn't absorb the following lines when rendered